app/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,10 @@
     <uses-permission android:name="android.permission.GET_ACCOUNTS" />
     <uses-permission android:name="android.permission.USE_CREDENTIALS" />
     <uses-permission android:name="android.permission.MANAGE_ACCOUNTS" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
+    <uses-permission android:name="android.permission.READ_MEDIA_VIDEO"/>
+    <uses-permission android:name="android.permission.READ_MEDIA_AUDIO"/>
```

Comment:
We do not support video nor audio upload for now. Maybe these two permissions could be commented out for now, in order to not ask the user for too many permissions? Or are these actually somehow necessary to access images?

Reply:
removed those permissions 👍